src/i18n/resources/en.ts

```diff
@@ -130,10 +130,12 @@ export const en: LanguageResource = {
     nodeaddress: 'Node Address',
   },
   error: {
-    notfound: 'Not found',
+    notfound:
+      'Looks like you have lost your way, Netmaker has a route for that!',
     network: 'Could not reach server',
     tokenexpire: 'Session has expired',
     unauthorized: 'Unauthorized access attempted',
+    error: '404 Error',
```

Comment:
this should be error404 or something more descriptive

Reply:
going to remove it it isnt needed now must have missed it